mlrun/config.py

```diff
@@ -419,11 +419,12 @@
         "k8s_secrets_project_name": "-marketplace-secrets",
         "catalog_filename": "catalog.json",
         "default_source": {
-            # Set to false to avoid creating a global source (for example in a dark site)
+            # Set false to avoid creating a global source (for example in a dark site)
             "create": True,
             "name": "mlrun_global_hub",
             "description": "MLRun global function hub",
-            "url": "https://raw.githubusercontent.com/mlrun/marketplace",
+            "url": "https://raw.githubusercontent.com/mlrun/marketplace/master",
+            "object_type": "functions",
```

Comment:
are there any other options we are looking to support? not sure I understand the need for that param

Reply:
yes, `functions` is just one type of marketplace, in the future we want to support more kinds of items, like `projects` and etc

Reply:
and you would want only one object type option? or just 1 default?

Reply:
I don't sure I follow, each marketplace source has only one `object_type` and all the items from a single marketplace source are from this type